lomond/parser.py

```diff
@@ -214,14 +214,10 @@ class TestParser(Parser):
         def parse(self):
             data = yield self.read_until(b'\r\n\r\n')
             yield data
-            data = yield self.read(1)
-            yield data
-            data = yield self.read(2)
-            yield data
-            data = yield self.read(4)
-            yield data
-            data = yield self.read(2)
-            yield data
+            for x in (1, 2, 4, 2):
```

Comment:
Generally I would approve of such a change, but this code is a throwaway test for debugging, it doesn't run in normal usage. The repetition is somewhat intentional, it makes makes it clear at a glance how many bytes are being read while debugging.

Reply:
> 
> 
> Generally I would approve of such a change, but this code is a throwaway test for debugging, it doesn't run in normal usage. The repetition is somewhat intentional, it makes makes it clear at a glance how many bytes are being read while debugging.

Fair enough :)